raiden/tests/unit/test_pfs_integration.py

```diff
@@ -78,6 +96,22 @@ def create_square_network_topology(
     return new_state, [address1, address2, address3, address4], channels
 
 
+PFS_CONFIG = PFSConfig(
+    info=PFSInfo(
+        url="abc",
+        price=TokenAmount(12),
+        chain_id=ChainID(42),
+        token_network_registry_address=factories.make_token_network_address(),
+        payment_address=factories.make_address(),
+        message="",
+        operator="",
+        version="",
+        settings="",
+    ),
+    maximum_fee=TokenAmount(100),
+    iou_timeout=BlockNumber(100),
+    max_paths=5,
+)
 CONFIG = {
     "services": {
```

Comment:
Looks like the `"services"` key is now unused and can be removed.

Reply:
Nice!